app/src/main/kotlin/com/wire/android/util/WireConstants.kt

```diff
@@ -0,0 +1,41 @@
+package com.wire.android.util
+
+import com.wire.kalium.logic.configuration.ServerConfig
+
+object WireConstants {
+    const val FB_STAGING_PROJECT_ID = "wiregcmpushstaging"
```

Comment:
FB stands for Firebase ? Could you use the full name FIREBASE_STAGING_PROJECT_ID ?

Reply:
okay will change it